kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -15,7 +15,13 @@
           />
         </template>
         <TextTruncatorCss
-          :text="resourceTitle"
+          v-if="windowBreakpoint <= 3"
+          :text="resourceTitle | truncateText(50)"
+          :maxLines="1"
+        />
+        <TextTruncatorCss
+          v-else
+          :text="resourceTitle | truncateText(70)"
```

Comment:
More out of curiosity, I'm wondering how the numbers 50 and 70 decided on? In the included picture, the `windowBreakpoint` is 6, this title has two more characters to display, and seems like it should fit, but it is truncated instead.
<img width="1438" alt="Screenshot 2023-08-30 at 2 05 31 PM" src="https://github.com/learningequality/kolibri/assets/46411498/3369fb1c-8781-439e-8a60-ff6fb3468e08">

Reply:
i have just added by manually  applying it by decreasing the size of window to various breakpoints for large screen when it is greater than 2 breakpoint means till 3 to 6 i have make 70.

it will show good in case of 6 breakpoint but when we take it to 3 or 4 any size of text above or about 75  can create problem of button overlap again.

we can also change it for breakpoint 5 and 6 as well seperately like about 80 to 85 (a it will be safe) but the inital thinking of mine is to cover approx 50-60 % of the header by title in case of breakpoint 5 and 6 .

Reply:
Thank you for the explanation, that makes sense to me! Now that all the tests pass, this should be good to merge in!